src/stories/components/breadcrumbs/customize-breadcrumbs/customize-breadcrums.tsx

```diff
@@ -0,0 +1,105 @@
+import styled from '@emotion/styled';
+import lightTheme from '@ses/styles/theme/light';
+import { Breadcrumbs } from '../breadcrumbs';
+
+interface Props {
+  items: {
+    label: string | JSX.Element;
+    url: string;
+  }[];
+  isLight: boolean;
+}
+
+const CustomizeBreadCrumbs = ({ items, isLight }: Props) => (
```

Comment:
This should be `CustomBreadCrumbs` right? "Customize" is an action

Reply:
you are right, done